bin/eslint-formatters-calypso/README.md

```diff
@@ -0,0 +1,82 @@
+# eslint-formatters-calypso
+
+A set of formatters to be used by ESLint and utilities to build new formatters upon.
+
+Formatters available:
+
+* `lines-modified`: downgrade ESLint `errors` to `warnings`, except for lines modified in the topic branch.
+* `lines-modified-and-specific-rules`: downgrade ESLint `errors` to `warnings`, except for lines modified in the topic branch and the rules in the `.eslines.json` file.
+* `parsing-errors`: takes the ESLint original report and outputs a new one containing only `parsing errors`.
+* `eslines`: a wrapper formatter. If the branch active is `master`, would return the results of `parsing-errors` formatter. Otherwise, would return the results of `lines-modified-and-specific-rules` formatter.
```

Comment:
Shouldn’t we also run all the enforced (`rulesToNotDowngrade`) rules in `master`, too?